Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_duration.py

```diff
@@ -411,7 +411,7 @@ def parse(self, source: ExtractResult, reference: datetime = None) -> Optional[D
         if reference is None:
             reference = datetime.now()
 
-        result = DateTimeParseResult(source)
+        value = None
```

Comment:
It is None because if the execution does not enter the if in which it is assigned a value, it would leave an error where value would not exist when it is used outside that if statement. So like this, if it does not enter in there, value will be None.